xapian-applications/omega/docs/postprocessing.rst

```diff
@@ -72,9 +76,9 @@ Log Data Format
       821f03288846297c2cf43c34766a38f7,54
       098f6bcd4621d373cade4e832627b4f6,42
 
-``search_log`` file: Much similar to ``search_log`` file but with an additional fifth field as follows:
+``final_log`` file: Similar to ``search_log`` file but with an additional fifth field as follows:
 
-  1. **QueryID** - an identifier for each query (generated using ``$hash`` OmegaScript command).
+  1. **QueryID** - an identifier for each query.
```

Comment:
You've removed the reference to `$hash{}` once; I noted it twice but it actually exists three times. Let's clean them all up, please.